charts/fairspace-keycloak/values.yaml

```diff
@@ -20,6 +20,7 @@ fairspaceKeycloak:
         cpu: "500m"
         memory: "2048Mi"
   postgresql:
+    password: ""
```

Comment:
Is there a reason to add empty password? Rather have a failing deployment, than default password ""

Reply:
let me see what else we can do

Reply:
okay, I've done it in a different way: it did not fail before my changes because of the default value in pipeline, but linter failed. I made it behaving the same adding brackets what allows to have nullable values. In this case, we will take the default value again if it is absent

Reply:
well, it does not improve, but at least it does not make it worse. I wouldn't spend time on that